src/net/uds/datagram.rs

```diff
@@ -135,18 +134,28 @@ impl UnixDatagram {
     }
 }
 
-impl Source for UnixDatagram {
-    fn register(&self, registry: &Registry, token: Token, interests: Interest) -> io::Result<()> {
+impl event::Source for UnixDatagram {
```

Comment:
`event::Source` is sort of special in this case, because `event` is part of the name as its called "event source" (not just "source"). But we don't want to write `use mio::event::EventSource`, so we `use mio::event` and then write `event::Source`. In almost all other cases we would `use` the entire part and just import the trait/struct etc.